File: NoOfPairs.java
/*package whatever //do not write package name here */

import java.util.*;
import java.lang.*;
import java.io.*;

class NoOfPairs {
	public static void main (String[] args) {
		Scanner s = new Scanner(System.in);
		int t = s.nextInt();
		while(t--!=0){
		    int n1 = s.nextInt();
		    int n2 = s.nextInt();
		    int arr1[] = new int[n1];
		    int arr2[] = new int[n2];
		    for(int i=0;i<n1;i++)
		        arr1[i] = s.nextInt();
		    for(int i=0;i<n2;i++)
		        arr2[i]= s.nextInt();
		        
		        int count=0;
		    for(int i=0;i<n1;i++){
		        for(int j=0;j<n2;j++){
		            if(Math.pow(arr1[i],arr2[j])>Math.pow(arr2[j],arr1[i])){
		                count++;
		            }
		        }
		    }
		    System.out.println(count);
		    
		}
	}
}

File: noOfPairs.cpp
#include <iostream>
#include<cmath>
using namespace std;

void countPairsBruteForce(int X[], int Y[], int m, int n) 
{ 
    int ans = 0; 
    for (int i = 0; i < m; i++) 
       for (int j = 0; j < n; j++) 
          if (pow(X[i], Y[j]) > pow(Y[j], X[i])) 
              ans++; 
    cout<<ans; 
} 

int main() {

		int t;
		cin>>t;
		while(t--!=0){
		    int n1;
		    cin>>n1;
		    int n2 ;
		    cin>>n2;
		    int arr1[n1];
		    int arr2[n2];
		    for(int i=0;i<n1;i++){
		        cin>>arr1[i]; 
            }
		    for(int i=0;i<n2;i++){
		        cin>>arr2[i]; 
            }
		        

		countPairsBruteForce(arr1,arr2,n1,n2);
        }
	return 0;
}